### PR TITLE
[CHK-2531] dispatcher get state polling

### PR DESCRIPTION
**Pull request by ciuffagianluca**
- Add NPG getState mock, with ok and ko result
- Fix properties names

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -21,7 +21,7 @@ PGS_PORT=8094
 
 WALLET_PORT=8095
 
-ECOMMERCE_EVENT_DISPATCHER_COMMIT_SHA=main
+ECOMMERCE_EVENT_DISPATCHER_COMMIT_SHA=CHK-2509-auth-requested-retry
```
Comment (pietro-tota) [inline .env:24]:
```suggestion
ECOMMERCE_EVENT_DISPATCHER_COMMIT_SHA=main
```

File: pagopa-ecommerce-event-dispatcher-service/event-dispatcher-service.env [hunk 2]
```diff
@@ -57,8 +57,9 @@ NOTIFICATION_RETRY_EVENT_MAX_ATTEMPTS=3
 PERSONAL_DATA_VAULT_API_KEY=personal-data-vault-api-key
 PERSONAL_DATA_VAULT_API_BASE_PATH="http://pagopa-pdv-mock:8080/tokenizer/v1/transaction/"
 
-AUTHORIZATION_REQUESTED_RETRY_EVENT_BASE_INTERVAL_SECONDS=600
-AUTHORIZATION_REQUESTED_RETRY_EVENT_MAX_ATTEMPTS=3
+AUTHORIZATION_OUTCOME_WAITING_EVENT_BASE_INTERVAL_SECONDS=60
+AUTHORIZATION_OUTCOME_WAITING_EVENT_MAX_ATTEMPTS=3
+AUTHORIZATION_OUTCOME_WAITING_EVENT_PAYMENT_TOKEN_VALIDITY_TIME_OFFSET=10
```
Comment (pietro-tota) [inline pagopa-ecommerce-event-dispatcher-service/event-dispatcher-service.env:62]: to be aligned to the event dispatcher used ones after name refactoring

**Review by pietro-tota — DISMISSED** [review]

File: pagopa-ecommerce-event-dispatcher-service/event-dispatcher-service.env (hunk 2 above)
Comment (ciuffagianluca) [reply pagopa-ecommerce-event-dispatcher-service/event-dispatcher-service.env:62]: These already are the name used in event dispatcher after refactoring

Comment (pietro-tota) [reply pagopa-ecommerce-event-dispatcher-service/event-dispatcher-service.env:62]: I was suggesting to change `AUTHORIZATION_OUTCOME_WAITING_EVENT_PAYMENT_TOKEN_VALIDITY_TIME_OFFSET ` adding the time unit in the env variable name to make it explicit. what are those? seconds, milliseconds?

**Review by pietro-tota — DISMISSED** [review]

**Pull request merged**